shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/data/mapper/OnUpdateFeedingPointToDomainMapper.kt

```diff
@@ -0,0 +1,26 @@
+package com.epmedu.animeal.feeding.data.mapper
+
+import OnUpdateFeedingPointSubscription.OnUpdateFeedingPoint
+import com.epmedu.animeal.feeding.domain.model.enum.AnimalState
+import com.epmedu.animeal.feeding.presentation.model.MapLocation
+import com.epmedu.animeal.foundation.switch.AnimalType
+import com.epmedu.animeal.feeding.domain.model.FeedingPoint as DomainFeedingPoint
+
+internal fun OnUpdateFeedingPoint.toDomainFeedingPoint() =
+    DomainFeedingPoint(
```

Comment:
There is a repeating code with OnCreateFeedingPoint.toDomainFeedingPoint() . What do you think if it is better to have a sealed class or interface as their parent and you will have to write converter extention only once for all the classes of the same type.

Reply:
How? These classes are generated by Apollo.

Reply:
My apologies, you are right, then it is not worth changing.